db-scripts/src/main/resources/migration.sql

```diff
@@ -380,4 +380,49 @@ ALTER TABLE `mutation` ADD COLUMN `DRIVER_FILTER_ANNOTATION` VARCHAR(80) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER` VARCHAR(50) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER_ANNOTATION` VARCHAR(80) NULL;
 
-UPDATE info SET DB_SCHEMA_VERSION="2.4.0";
+-- ========================== new reference genome gene related tables =============================================
```

Comment:
I think you are now just missing the 2.4.1 header so that the python migration script will work: